.gitignore

```diff
@@ -3,6 +3,8 @@
 /pkg
 /vendor
 envconsul
+config.hcl
```

Comment:
I don't mind adding these to the .gitignore but I prefer keeping unrelated changes out of feature PRs. Would you mind removing this? Please feel free to submit it as a standalone PR if you'd like (I'll just merge it).

Reply:
Fixed!